lib/helper/Puppeteer.js

```diff
@@ -1335,7 +1335,14 @@ class Puppeteer extends Helper {
    */
   async saveScreenshot(fileName, fullPage) {
     const fullPageOption = fullPage || this.options.fullPageScreenshots;
-    const outputFile = path.join(global.output_dir, fileName);
+    let outputFile;
+
+    if (!fileName.includes('/')) {
```

Comment:
copy-paste detected :)
let's move it somewhere to `utils`

Reply:
moved it to `utils`